### PR TITLE
Feature/abd content selection

### PR DESCRIPTION
**Pull request by Abdeboskey**
# Som Timer Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
* Create `ContentSelection` component
* Create initial unit tests and styling for `ContentSelection`
* Add mobile styling for `ContentSelection` as well

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
* The test for this component is located at `src/Components/ContentSelection/ContentSelection.test.js`
# Message for reviewer:
* We are waiting to do some more Icon R&D before putting icons on the buttons.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

Screen Shot 2020-10-25 at 2 04 38 PM

### CODE REVIEW

File: src/Components/ContentSelection/ContentSelection.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const ContentSelection = () => {
+  return (
+    <section className={style.ContentSelection}>
+      <h2 className={style.prompt}>How would you like to spend your break?</h2>
+      <div className={style.options}>
+        <div>
+          <button 
+            className={style.button} 
+            aria-label="somatic exercise"
```
Comment (rwilliams659) [inline src/Components/ContentSelection/ContentSelection.js:12]: Love the attention given to accessibility with the aria labels

File: src/Components/ContentSelection/ContentSelection.module.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  box-shadow: 0px 0px 10px $darkBlue;
+  margin: 2em 5.5em 0 5.5em;
+
+  &:hover {
+    box-shadow: 0px 0px 30px $lightBlueAccent;
+    cursor: pointer;
+  }
+}
+
+@media screen and (max-width: 688px) {
```
Comment (rwilliams659) [inline src/Components/ContentSelection/ContentSelection.module.scss:38]: Also love the media queries, great responsiveness here!

File: src/Components/ContentSelection/ContentSelection.module.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    flex-direction: column;
+    margin-top: 1em;
+  }
+}
+
+@media screen and (max-height: 740px) {
+
+  .button {
+    @include size(6.5em, 6.5em);
+  }
```
Comment (jkwest-93) [inline src/Components/ContentSelection/ContentSelection.module.scss:54]: Great work with responsive design here!

**Review by rwilliams659 — APPROVED** [review]
Looks awesome, again can't wait to see what icons we come up with!

File: src/Components/ContentSelection/ContentSelection.test.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  it('should display the correct buttons', () => {
+    const { getByRole } = render(<ContentSelection />)
+
+    const yogaMovementButton = getByRole('button', { name: /yoga\/movement/i })
+    const somaticExerciseButton = getByRole('button', { name: /somatic exercise/i })
+    const breathworkMeditationButton = getByRole('button', { name: /breathwork\/meditation/i })
+
+    expect(yogaMovementButton).toBeInTheDocument()
+    expect(somaticExerciseButton).toBeInTheDocument()
+    expect(breathworkMeditationButton).toBeInTheDocument()
```
Comment (jkwest-93) [inline src/Components/ContentSelection/ContentSelection.test.js:25]: Curious about Eric's feedback on bundling our unit tests. Wondering if we should combine these two...

**Review by jkwest-93 — APPROVED** [review]

File: src/Components/ContentSelection/ContentSelection.test.js (hunk 4 above)
Comment (Abdeboskey) [reply src/Components/ContentSelection/ContentSelection.test.js:25]: I hath not yet asked this Q, I'll send him a note about it!

**Review by Chulstro — APPROVED** [review]
Checklist looks good

File: src/Components/ContentSelection/ContentSelection.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+          ></button>
+          <p>Somatic Exercise</p>
+        </div>
+        <div>
+          <button
+            className={style.button}
+            aria-label="breathwork/meditation"
```
Comment (sienna-kopf) [inline src/Components/ContentSelection/ContentSelection.js:19]: The inclusion of these aria-labels is really really cool.

**Review by sienna-kopf — APPROVED** [review]

File: src/Components/ContentSelection/ContentSelection.js (hunk 1 above)
Comment (sciencefixion) [reply src/Components/ContentSelection/ContentSelection.js:12]: +1 to what @rwilliams659 said! I love that accessibility is a priority for this project.

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**